server/isomorphic-routing/index.js

```diff
@@ -39,7 +34,8 @@ export function serverRouter( expressApp, setUpRoute, section ) {
 					setRouteMiddleware,
 					...middlewares
 				),
-				serverRender
+				serverRender,
+				serverRenderError
```

Comment:
I don't think `serverRenderError` should be called here -- it's the `if` branch that takes care of errors, this one is for "successful" cases.